Gruntfile.js

```diff
@@ -571,35 +570,26 @@ module.exports = function(grunt) {
     'bundle',
   ]);
 
-  grunt.registerTask('linux-prod', ['linux-prod-package', 'electronbuilder:linux_prod']);
-
-  grunt.registerTask('linux-custom', [
+  grunt.registerTask('linux-custom-package', [
     'clean:wrap',
     'update-keys',
     'gitinfo',
     'set-custom-data',
     'release-custom',
     'bundle',
-    'electronbuilder:linux_prod',
   ]);
 
-  grunt.registerTask('linux-other-internal', [
-    'clean:wrap',
-    'update-keys',
-    'gitinfo',
-    'set-custom-data',
-    'release-internal',
-    'bundle',
-    'electronbuilder:linux_other',
-  ]);
+  grunt.registerTask('linux', ['linux-package', 'electronbuilder:linux_internal']);
 
-  grunt.registerTask('linux-other', [
-    'clean:wrap',
-    'update-keys',
-    'gitinfo',
-    'set-custom-data',
-    'release-prod',
-    'bundle',
-    'electronbuilder:linux_other',
-  ]);
+  grunt.registerTask('linux-prod', ['linux-prod-package', 'electronbuilder:linux_prod']);
+
+  grunt.registerTask('linux-custom', ['linux-custom-package', 'electronbuilder:linux_prod']);
+
+  grunt.registerTask('linux-other-internal-package', ['linux-package']);
+
+  grunt.registerTask('linux-other-internal', ['linux-other-internal-package', 'electronbuilder:linux_other']);
+
+  grunt.registerTask('linux-other-package', ['linux-prod-package']);
```

Comment:
Note that `linux-other-internal-package` and `linux-other-package` are now simply aliases to other tasks, I didn't remove them because I don't know if you use them or not, but they are potential candidates for removal.

Reply:
No silly me, of course the only usage is right here because `-package` didn't exist before 😄  Updated.